src/app/app.routing.ts

```diff
@@ -34,8 +34,13 @@ import { StarredEntriesComponent } from './starredentries/starredentries.compone
 export const CLIENT_ROUTER_PROVIDERS = [AuthGuard];
 
 const APP_ROUTES: Routes = [
-  { path: '', component: HomePageComponent, pathMatch: 'full', data: { title: 'Dockstore' } },
-  { path: 'beta-homepage', component: HomeLoggedInComponent, pathMatch: 'full', data: { title: 'Dockstore' } },
+  { path: '', component: HomeComponent, pathMatch: 'full', data: { title: 'Dockstore' } },
+  {
+    path: 'dashboard',
+    component: HomeLoggedInComponent,
+    pathMatch: 'full',
+    data: { title: 'Dockstore' },
```

Comment:
I suggest `My Dockstore`, or something different from the other page.

Reply:
Yes! I missed that :)

Reply:
> Would `Dockstore | Dashboard` make more sense?

That works for me!